Лабораторная работа №4 (с подзапросами)

shappovalovv opened the pull request:
Отправляю на проверку последнюю лабораторную работу

shappovalovv committed:
Лабораторная работа №4 (с подзапросами)

Отправляю на проверку последнюю лабораторную работу

MarinaPaley commented on Лабораторные/lab4.sql:
```diff
@@ -0,0 +1,53 @@
+      WHERE ListPrice > 0
+    ) AS T
+  )
+EXCEPT
```
А это зачем?

MarinaPaley reviewed (changes requested):
А можно ещё и текст вопросов, пожалуйста?)

shappovalovv committed:
Исправления по четвертой лабораторной работе

В файл .sql было добавлено описание к каждому заданию;
Исправлено пятое задание.

shappovalovv committed:
.

shappovalovv committed:
.
